onthequad/onthequad/dummyevents.py

```diff
@@ -1,72 +1,63 @@
 import falcon
 import json
+from .DataBaseCalls import DataBaseCalls
 
 
 class DumbEvents:
 
     def on_get(self, req, resp):
         # data = json.loads(req.stream.read())
-        sendback = {
-            "id": 987654,
-            "name": "Volleyball",
-            "location": "Volleyball court",
-            "date": "02/14/2019",
-            "starttime": 1315,
-            "endtime": 1430,
-            "category": "Sports and Games",
-            "creator": {
-                "id": 12345,
-                "firstname": "John",
-                "lastname": "Smith",
-                "email": "jsmith@msudenver.edu",
-                "username": "jsmith",
-                "photoURL": "",
-                "events": []
-            },
-            "attendees": [],
-            "availablespots": 10,
-            "coordinates": "",
-            "public": True
-        }
-
-        resp.body = json.dumps(sendback)
+        send = DataBaseCalls.getAllEvents()
+        resp.body = json.dumps(send)
         resp.status = falcon.HTTP_200
 
     def on_post(self, req, resp):
         data = json.loads(req.stream.read())
-        if data['method'] == 'OK':
+        # this logic can be removed since front end will input validation
+        if any(data):
+            send = {
+                'Message' : 'Event has been created'
+            }
+            DataBaseCalls.storeEvent(data)
+            resp.body = json.dumps(send)
             resp.status = falcon.HTTP_200
         else:
+            send = {
+                'Message' : 'Cannot create empty event'
+            }
+            resp.body = json.dumps(send)
             resp.status = falcon.HTTP_404
 
     def on_delete(self, req, resp):
         data = json.loads(req.stream.read())
-
-        if data['method'] is None:
-            sendback = {
-                'msg': "there is nothing to delete"
+        eventID = next(iter(data))
+        if eventID is None:
+            send = {
+                'Message' : 'Event cannot be found'
             }
-            resp.body = json.dumps(sendback)
-            resp.status = falcon.HTTP_400
+            resp.body = json.dumps(send)
+            resp.status = falcon.HTTP_404
```

Comment:
We need to change this from a 404 to a 400 status code. A 404 is "Server not found" while a 400 is "Bad Request"